src/OVAL/probes/probe/offline_mode.c

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file   offline_mode.c
+ * @brief  file containg the dummy probe_offline_mode function
+ */
+
+/*
+ * Copyright 2016 Red Hat Inc., Durham, North Carolina.
+ * All Rights Reserved.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ */
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include <probe-api.h>
+
+/**
+ * Dummy probe_preload function
+ * Return type of supported offline mode.
```

Comment:
Most likely you have forgotten to remove this comment. This function isn't `probe_preload` and doesn't return anything.